### PR TITLE
edit index

### PR DESCRIPTION
**Pull request by M-Khairy-B**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -30,7 +30,7 @@
             <img src="img/gamer 1.png" alt="" />
```
Comment (mukhtar-ali) [inline index.html:30]: Header padding is wrong

File: index.html [hunk 2]
```diff
@@ -97,7 +100,7 @@ <h1>GAMERS</h1>
     </nav>
     <header class="pb-4">
```
Comment (mukhtar-ali) [inline index.html:101]: Wrong usage of HTML tags

File: index.html (hunk 1 above)
Comment (mukhtar-ali) [inline index.html:30]: Why did you use two google fonts imports?

Comment (mukhtar-ali) [inline index.html:30]: Missing important meta tags in the head tag

File: index.html [hunk 3]
```diff
@@ -30,7 +30,7 @@
             <img src="img/gamer 1.png" alt="" />
           </div>
           <div>
-            <h1>GAMERS</h1>
+            <h1>GAMERS  </h1>
```
Comment (mukhtar-ali) [inline index.html:33]: Logo is not like the design.
I already told you to take it as SVG or PNG, either do this or fix it to be like the design

File: index.html [hunk 4]
```diff
@@ -70,10 +70,13 @@ <h1>GAMERS</h1>
...
-                <a class="nav-link" href="#">Shop</a>
+                <a class="nav-link" href="shipping.html">Shipping</a>
               </li>
               <li class="nav-item">
-                <a class="nav-link" href="#">Contact</a>
+                <a class="nav-link" href="contact.html">Contact</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="client.html">About</a>
               </li>
```
Comment (mukhtar-ali) [inline index.html:80]: font is not like the design

File: index.html [hunk 5]
```diff
@@ -70,10 +70,13 @@ <h1>GAMERS</h1>
                 <a class="nav-link" href="cart.html">Cart </a>
```
Comment (mukhtar-ali) [inline index.html:70]: Arrow beside the "Home" is not like the design

File: index.html [hunk 6]
```diff
@@ -70,10 +70,13 @@ <h1>GAMERS</h1>
...
               </li>
               <li class="nav-item">
-                <a class="nav-link" href="#">Contact</a>
+                <a class="nav-link" href="contact.html">Contact</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="client.html">About</a>
               </li>
             </ul>
           </div>
```
Comment (mukhtar-ali) [inline index.html:82]: Search is not like the design

File: index.html [hunk 7]
```diff
@@ -97,7 +100,7 @@ <h1>GAMERS</h1>
     </nav>
     <header class="pb-4">
       <div class="container">
-        <div class="row">
+        <div class="row ">
           <div class="col-md-6 d-flex flex-column text-contorller">
             <h2>PS5 V2 <br />DualShock 5</h2>
             <p>
```
Comment (mukhtar-ali) [inline index.html:106]: Revisit the design

**Review by mukhtar-ali — CHANGES_REQUESTED** [review]

File: index.html (hunk 7 above)
Comment (mukhtar-ali) [inline index.html:106]: Buy Now button, wrong alignment

Comment (mukhtar-ali) [inline index.html:106]: Buy Now button, wrong padding

Comment (mukhtar-ali) [inline index.html:106]: Buy Now button, wrong font size

Comment (mukhtar-ali) [inline index.html:106]: Slider dots position is wrong

Comment (mukhtar-ali) [inline index.html:106]: Slider dots, not like the design

Comment (mukhtar-ali) [inline index.html:106]: "Featured Products" title is not like the design

Comment (mukhtar-ali) [inline index.html:106]: "Featured Products" padding is not like the design

Comment (mukhtar-ali) [reply index.html:106]: PS: Section padding are given to the section element not it's children

Comment (mukhtar-ali) [inline index.html:106]: "Featured Products" title, You should use margin instead of padding to make space between elements

Comment (mukhtar-ali) [inline index.html:106]: Re-visit "Featured Products" section as it's not like the design

Comment (mukhtar-ali) [inline index.html:106]: Re-visit "Featured Products" section dots as they are not like the design

File: index.html (hunk 1 above)
Comment (mukhtar-ali) [inline index.html:30]: Work on the responsive.

File: index.html (hunk 7 above)
Comment (mukhtar-ali) [inline index.html:106]: space between "Latest Products" and tabs are not like the design

Comment (mukhtar-ali) [inline index.html:106]: "Latest Products" Tabs are not like the design.

Comment (mukhtar-ali) [inline index.html:106]: "Latest Products" => Space between tabs and cards are not like the design

Comment (mukhtar-ali) [inline index.html:106]: Each card content should have paddings to separate the content from the edges, the card content padding is missing and the space between the price and bottom of the card is not like the design

File: index.html [hunk 8]
```diff
@@ -539,7 +542,7 @@ <h4>XBOX SERIES X CONTROLLER</h4>
           <h2>What Gamers Offer!</h2>
         </div>
         <div class="row">
-          <div class="col-3">
+          <div class="col-md-3">
             <div class="layer-2 text-center">
               <img src="img/free-delivery 1.png" alt="" />
               <h6 class="my-3">Free Shiping</h6>
```
Comment (mukhtar-ali) [inline index.html:548]: Padding left and right is not like the design

File: index.html [hunk 9]
```diff
@@ -539,7 +542,7 @@ <h4>XBOX SERIES X CONTROLLER</h4>
           <h2>What Gamers Offer!</h2>
```
Comment (mukhtar-ali) [inline index.html:542]: Don't use padding to make outer space, please fix in all titles

**Review by mukhtar-ali — CHANGES_REQUESTED** [review]

File: index.html [hunk 10]
```diff
@@ -569,7 +572,7 @@ <h6 class="my-3">Quality Product</h6>
               </p>
             </div>
           </div>
-          <div class="col-3">
+          <div class="col-md-3">
             <div class="layer-2 text-center">
               <img src="img/24-hours-support 1.png" alt="" />
               <h6 class="my-3">24/7 Support</h6>
```
Comment (mukhtar-ali) [inline index.html:578]: "Trending Products" Card issue same as the latest products

Comment (mukhtar-ali) [inline index.html:578]: Latest Blog title is wrong

Comment (mukhtar-ali) [inline index.html:578]: Latest Blog spacing issues

Comment (mukhtar-ali) [inline index.html:578]: The blog card have a lot of issues

- Pink one is an hover state.
- There should be anchors in the following: 
     1- Image
     2- Title
     3- Read More
- Content padding should be handled from one place, you shouldn't create padding for each element
- Image doesn't have any constrains, meaning if the user added a bigger image that will break your design.

Comment (mukhtar-ali) [inline index.html:578]: Footer not like the design

**Review by mukhtar-ali — CHANGES_REQUESTED** [review]

File: index.html (hunk 10 above)
Comment (mukhtar-ali) [inline index.html:578]: All the anchors in the footer are not functional

Comment (mukhtar-ali) [inline index.html:578]: Missing form tag from the newsletter

File: category.html [hunk 11]
```diff
@@ -66,10 +66,13 @@ <h1>GAMERS</h1>
                 <a class="nav-link" href="cart.html">Cart </a>
               </li>
               <li class="nav-item">
```
Comment (mukhtar-ali) [inline category.html:68]: Card title is not like the design

Comment (mukhtar-ali) [inline category.html:68]: The cards should be clickable to lead to the category page!!!

File: index.html (hunk 1 above)
Comment (mukhtar-ali) [inline index.html:30]: Missing favicon

File: Products.html [hunk 12]
```diff
@@ -68,10 +68,13 @@ <h1>GAMERS</h1>
...
               </li>
               <li class="nav-item">
-                <a class="nav-link" href="#">Contact</a>
+                <a class="nav-link" href="contact.html">Contact</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="client.html">About</a>
               </li>
             </ul>
           </div>
```
Comment (mukhtar-ali) [inline Products.html:80]: Re-visit product card design, as it is not like the design

Comment (mukhtar-ali) [inline Products.html:80]: Space between the cards and the row above are not like the design

File: Products.html [hunk 13]
```diff
@@ -68,10 +68,13 @@ <h1>GAMERS</h1>
...
-                <a class="nav-link" href="#">Shop</a>
+                <a class="nav-link" href="shipping.html">Shipping</a>
               </li>
               <li class="nav-item">
-                <a class="nav-link" href="#">Contact</a>
+                <a class="nav-link" href="contact.html">Contact</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="client.html">About</a>
               </li>
```
Comment (mukhtar-ali) [inline Products.html:78]: Sort by is not centered as the design

File: details.html [hunk 14]
```diff
@@ -68,10 +68,13 @@ <h1>GAMERS</h1>
                 <a class="nav-link" href="cart.html">Cart </a>
```
Comment (mukhtar-ali) [inline details.html:68]: Slider is missing

File: details.html [hunk 15]
```diff
@@ -68,10 +68,13 @@ <h1>GAMERS</h1>
...
               </li>
```
Comment (mukhtar-ali) [inline details.html:69]: Product stars are not like the design

File: details.html [hunk 16]
```diff
@@ -68,10 +68,13 @@ <h1>GAMERS</h1>
...
               <li class="nav-item">
```
Comment (mukhtar-ali) [inline details.html:70]: Add to cart & buy now are not like the design

File: details.html [hunk 17]
```diff
@@ -68,10 +68,13 @@ <h1>GAMERS</h1>
...
-                <a class="nav-link" href="#">Shop</a>
```
Comment (mukhtar-ali) [inline details.html:71]: Description & additional info tabs are not working

File: details.html [hunk 18]
```diff
@@ -68,10 +68,13 @@ <h1>GAMERS</h1>
...
+                <a class="nav-link" href="shipping.html">Shipping</a>
```
Comment (mukhtar-ali) [inline details.html:71]: Revisit related products

File: cart.html [hunk 19]
```diff
@@ -68,10 +68,13 @@ <h1>GAMERS</h1>
                 <a class="nav-link" href="#">Cart </a>
               </li>
```
Comment (mukhtar-ali) [inline cart.html:69]: Revisit cart design

File: cart.html [hunk 20]
```diff
@@ -68,10 +68,13 @@ <h1>GAMERS</h1>
...
               <li class="nav-item">
-                <a class="nav-link" href="#">Shop</a>
```
Comment (mukhtar-ali) [inline cart.html:71]: Missing shipping page

File: cart.html [hunk 21]
```diff
@@ -68,10 +68,13 @@ <h1>GAMERS</h1>
...
+                <a class="nav-link" href="shipping.html">Shipping</a>
```
Comment (mukhtar-ali) [inline cart.html:71]: Missing order is completed page

File: cart.html [hunk 22]
```diff
@@ -68,10 +68,13 @@ <h1>GAMERS</h1>
...
               </li>
```
Comment (mukhtar-ali) [inline cart.html:72]: Missing contact page

File: cart.html [hunk 23]
```diff
@@ -68,10 +68,13 @@ <h1>GAMERS</h1>
...
               <li class="nav-item">
```
Comment (mukhtar-ali) [inline cart.html:73]: Missing client page

File: cart.html [hunk 24]
```diff
@@ -68,10 +68,13 @@ <h1>GAMERS</h1>
...
-                <a class="nav-link" href="#">Contact</a>
+                <a class="nav-link" href="contact.html">Contact</a>
```
Comment (mukhtar-ali) [inline cart.html:74]: Missing login page

**Review by mukhtar-ali — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Ahmed-Hassan-96) [reply index.html:30]: Not done

File: index.html (hunk 2 above)
Comment (Ahmed-Hassan-96) [reply index.html:101]: Not done

File: index.html (hunk 3 above)
Comment (Ahmed-Hassan-96) [reply index.html:33]: Not done

File: index.html (hunk 4 above)
Comment (Ahmed-Hassan-96) [reply index.html:80]: spaces between links not like the design

File: index.html (hunk 5 above)
Comment (Ahmed-Hassan-96) [reply index.html:70]: Not done

File: index.html (hunk 6 above)
Comment (Ahmed-Hassan-96) [reply index.html:82]: Not done

File: index.html (hunk 1 above)
Comment (M-Khairy-B) [reply index.html:30]: Done

File: index.html (hunk 2 above)
Comment (M-Khairy-B) [reply index.html:101]: Done

File: index.html (hunk 1 above)
Comment (M-Khairy-B) [reply index.html:30]: Done

File: index.html (hunk 3 above)
Comment (M-Khairy-B) [reply index.html:33]: Done

File: index.html (hunk 4 above)
Comment (M-Khairy-B) [reply index.html:80]: Done

File: index.html (hunk 5 above)
Comment (M-Khairy-B) [reply index.html:70]: Done

File: index.html (hunk 6 above)
Comment (M-Khairy-B) [reply index.html:82]: Done

File: index.html (hunk 7 above)
Comment (M-Khairy-B) [reply index.html:106]: Done

Comment (M-Khairy-B) [reply index.html:106]: Done

Comment (M-Khairy-B) [reply index.html:106]: Done

Comment (M-Khairy-B) [reply index.html:106]: Done

File: index.html (hunk 1 above)
Comment (M-Khairy-B) [reply index.html:30]: Done

File: index.html (hunk 7 above)
Comment (M-Khairy-B) [reply index.html:106]: Done

Comment (M-Khairy-B) [reply index.html:106]: done

Comment (M-Khairy-B) [reply index.html:106]: Done

Comment (M-Khairy-B) [reply index.html:106]: Done

Comment (M-Khairy-B) [reply index.html:106]: Done

Comment (M-Khairy-B) [reply index.html:106]: Done

File: index.html (hunk 8 above)
Comment (M-Khairy-B) [reply index.html:548]: Done

File: index.html (hunk 9 above)
Comment (M-Khairy-B) [reply index.html:542]: Done

File: index.html (hunk 10 above)
Comment (M-Khairy-B) [reply index.html:578]: Done

Comment (M-Khairy-B) [reply index.html:578]: done

Comment (M-Khairy-B) [reply index.html:578]: done

Comment (M-Khairy-B) [reply index.html:578]: Done

File: index.html (hunk 7 above)
Comment (M-Khairy-B) [reply index.html:106]: done

File: index.html (hunk 10 above)
Comment (M-Khairy-B) [reply index.html:578]: done

File: category.html (hunk 11 above)
Comment (M-Khairy-B) [reply category.html:68]: done

File: index.html (hunk 1 above)
Comment (M-Khairy-B) [reply index.html:30]: done

File: index.html (hunk 10 above)
Comment (M-Khairy-B) [reply index.html:578]: done

File: category.html (hunk 11 above)
Comment (M-Khairy-B) [reply category.html:68]: done

File: cart.html (hunk 20 above)
Comment (M-Khairy-B) [reply cart.html:71]: done

**Pull request closed without merging**